packages/utilities/psammead-styles/README.md

```diff
@@ -0,0 +1,25 @@
+# psammead-styles &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-styles.svg)](https://www.npmjs.com/package/@bbc/psammead-styles) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+## Usage
+
+This package is a collection of variable constants which can be imported into your application.
+
```

Comment:
Would be good to add a line with `npm install @bbc/psammead-styles` before this step.